web/styles/zulip.css

```diff
@@ -1006,19 +1006,6 @@ strong {
     word-wrap: break-word;
 }
 
-#message_edit_tooltip {
-    float: right;
```

Comment:
I think removing this `float: right` broke the alignment of the text, making it feel adrift in your screenshot.

Reply:
It seems like that in the in the screenshot because of the tooltip but the alignment is fine actually. I have attached the screenshot without the tooltip  for review:
![text](https://github.com/zulip/zulip/assets/119798962/fda0dab7-fb97-4d55-9640-ba06f29b5ee1)

Reply:
Ahh, OK.